### DISCUSSION TITLE
Make loose equipment a hazard

### DISCUSSION THREAD

**Discussion by Hinny** [FakeFishGames/Barotrauma#9455] [Ideas and suggestions]
To increase the incentive to place crates, ammo boxes and railgun shells in their respective racks, it would be cool if these can collide with crew to injure them and maybe do some small bit of hull damage in case of a sub collision. It feels quite dangerous to have these kinds of items just lying around loose in the sub.

Maybe crates etc can also spill out their contents if hit hard enough, while not in a rack.

This could be expanded to many types of smaller items as well (but much less damage) if you want.

**Comment by MaaiG** [comment]
>maybe do some small bit of hull damage in case of a sub collision.

I don’t know about small items, but ammo, explosives, and [depth charges](https://github.com/Regalis11/Barotrauma/discussions/7461#discussioncomment-2928379)... definitely!

**Comment by ubertpendragon** [comment]
I managed to make ammo crates deal damage when they impact something solid near characters
https://user-images.githubusercontent.com/73229309/175379153-231a73d0-465c-4537-ba41-7167d85adb28.mp4

Ideally the boxes should impact the character themselves but I'm not sure how to do that

**Reply by MaaiG** [reply]
> I managed to make ammo crates deal damage

Looks very dangerous! 👀

**Comment by Ek-Videogames** [comment]
Add more ways to secure equipment.

**Reply by ubertpendragon** [reply]
Aren't crate/ammo shelves enough?

**Reply by Ek-Videogames** [reply]
Not subs that don't have them - but I guess that can be addressed in design tickets.

**Comment by iridescent-lightning** [comment]
Barotrauma Die Hard has item collision
